app/mailers/report_mailer.rb

```diff
@@ -1,21 +1,59 @@
 # frozen_string_literal: true
 # rubocop:disable Metrics/MethodLength
 class ReportMailer < ApplicationMailer
+  def set_headers
+    # Disable SendGrid's clicktracking, it creates ugly URLs.
+    # See: https://sendgrid.com/docs/API_Reference/SMTP_API/apps.html
+    headers['X-SMTPAPI'] =
+      '{ "filters" : { "clicktrack" : { "settings" : { "enable" : 0 } } } }'
+  end
+
+  def project_info_url(id)
+    ('https://' + (ENV['PUBLIC_HOSTNAME'] || 'localhost') +
+      '/projects/' + id.to_s).freeze
+  end
+
+  # Report to Linux Foundation that a project's status has changed.
   def project_status_change(project, old_badge_status, new_badge_status)
     @project = project
     @old_badge_status = old_badge_status
     @new_badge_status = new_badge_status
-    @project_info_url = ('https://' + (ENV['PUBLIC_HOSTNAME'] || 'localhost') +
-                        '/projects/' + @project.id.to_s).freeze
+    @project_info_url = project_info_url(@project.id)
     @report_destination = 'cii-badge-log@lists.coreinfrastructure.org'
-    # Disable SendGrid's clicktracking, it creates ugly URLs.
-    # See: https://sendgrid.com/docs/API_Reference/SMTP_API/apps.html
-    headers['X-SMTPAPI'] =
-      '{ "filters" : { "clicktrack" : { "settings" : { "enable" : 0 } } } }'
+    set_headers
     mail(
       to: @report_destination,
       subject: "Project #{project.id} status change to " \
                         "passing=#{new_badge_status}"
     )
   end
+
+  def subject_for(new_badge_status)
+    case new_badge_status
+    when 'passing'
+      'CONGRATUlATIONS on achieving a passing best practices badge!'
+    else
+      return 'Your best practices badge is no longer passing'
+    end
+  end
```

Comment:
Okay. We'll have to revisit this once we add new levels, but that's in the future.